atoma-proxy/src/server/middleware.rs

```diff
@@ -1215,20 +1214,19 @@ pub(crate) mod utils {
                 },
             )
             .map_err(|e| AtomaProxyError::InternalError {
-                message: format!("Failed to send GetNodePublicAddress event: {}", e),
+                message: format!("Failed to send GetNodePublicAddress event: {e:?}"),
```

Comment:
Do we need the `:?`?

Reply:
Yes see https://github.com/atoma-network/atoma-proxy/pull/175#discussion_r1936051622